Gruntfile.js

```diff
@@ -36,7 +36,7 @@ module.exports = function(grunt) {
         sourceMap: true,
         // This controls the compiled css and can be changed to nested, compact or compressed.
         outputStyle: 'expanded',
-        precision: 5
+        precision: 10
```

Comment:
Technically bootstraps need 8, but it doesn't hurt to go to 10. most browsers ignore past a certain threshold anyway.